### PR TITLE
[#31][#32] [UI] Star Emoji answer

### LINKED ISSUE #31: [UI] As a user, I can see the star question when taking a survey
## Why

App must display the star question before taking a survey

## Acceptance Criteria

- Must display star question
- When star is tapped, it and all stars to the left must be selected
  - All stars to the right must be unselected
- All unselected stars must be faded out 

## Resources

- [Design](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=2632%3A305)

### LINKED ISSUE #32: [Integrate] As a user, I can see the star question when taking a survey
## Why

App must display the star question before taking a survey

## Acceptance Criteria

- Must display star question
- Must display question with `displayType` "star"

## Resources

- [Design](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=2632%3A305)

### PR DESCRIPTION
**Pull request by lydiasama**
- Closes https://github.com/nimblehq/flutter-ic-lydia-ryan/issues/31
- Closes https://github.com/nimblehq/flutter-ic-lydia-ryan/issues/32

## What happened 👀

- Implement UI for emoji answer 👉🏻  focus on `Star` ⭐  type

## Insight 📝

- There is the potential that the answer contents will have a different height because the question titles are different on each question.

## Proof Of Work 📹

https://github.com/nimblehq/flutter-ic-lydia-ryan/assets/12026942/3bdeff26-512e-4010-a003-c2fd1f8868ad

### CODE REVIEW

File: lib/screens/surveydetails/widget/question_paging_widget.dart [hunk 1]
```diff
@@ -158,6 +161,29 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
...
+  Widget _buildAnswerWidget(QuestionUiModel question) {
+    switch (question.displayType) {
+      case DisplayType.star:
+        return _answerEmojiRatingWidget(question, '⭐️');
+      default:
+        return const SizedBox();
+    }
+  }
+
+  Widget _answerEmojiRatingWidget(QuestionUiModel question, String emoji) {
```
Comment (ryan-conway) [inline lib/screens/surveydetails/widget/question_paging_widget.dart]: I think the convention is to use a verb at the start of the name 😇 
```suggestion
  Widget _buildAnswerEmojiRatingWidget(QuestionUiModel question, String emoji) {
```

File: lib/screens/surveydetails/widget/question_paging_widget.dart [hunk 2]
```diff
@@ -158,6 +161,29 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
...
+    return AnswerEmojiRatingWidget(
+      emoji: emoji,
+      count: question.answers.length,
+      onRated: (int rating) {},
```
Comment (ryan-conway) [inline lib/screens/surveydetails/widget/question_paging_widget.dart]: Shall we add a `TODO` here to save the answer? 😄

**Review by ryan-conway — COMMENTED** [review]
The rest LGTM 👍🏻

**Review by ryan-conway — APPROVED** [review]
LGTM 🚀

**Review by manh-t — APPROVED** [review]
LGTM :clap:

**Pull request merged**